eslint/eslintrc-frontend-shared.yml

```diff
@@ -0,0 +1,33 @@
+# Don't inherit settings from e.g. ~.
+root: true
+
+env:
+    browser: true
+
+plugins:
+    - "react"
+
+parserOptions:
+    # Allow JSX.
+    ecmaFeatures:
+        jsx: true
+
+# These are defined in dev-tools/Gruntfile-shared.js in the front-end build
+# system.
+globals:
+    __project_root: true
+    __node_modules: true
+    __dev_tools: true
+
+extends:
+    "./eslintrc-shared.yml"
+
+rules:
+    # 2 means error, 1 means warning, 0 means ignore.
+
+    # JSX.
+    jsx-quotes: 2
+
```

Comment:
Now that this is separated out, can you mention `eslint-plugin-react` is required, and include a link?